src/js/builtins.d.ts

```diff
@@ -363,6 +363,7 @@ declare function $isAbortSignal(signal: unknown): signal is AbortSignal;
 declare function $isAbsolute(): TODO;
 declare function $isDisturbed(): TODO;
 declare function $isPaused(): TODO;
+/** FIXME: does not exist. */
 declare function $isWindows(): TODO;
```

Comment:
this can be safely be deleted. i'm not sure why it was added. the correct way to do platform checks is via the idomatic `process.platform === 'win32'`, which will be tree-shaked.